crates/pallet-subspace/Cargo.toml

```diff
@@ -40,6 +40,7 @@ subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primiti
 subspace-erasure-coding = { version = "0.1.0", path = "../subspace-erasure-coding" }
 subspace-farmer-components = { version = "0.1.0", path = "../subspace-farmer-components" }
 subspace-kzg = { version = "0.1.0", path = "../../shared/subspace-kzg" }
+subspace-logging = { version = "0.0.1", path = "../../shared/subspace-logging" }
```

Comment:
I don’t think we need to add this dependency here, because it isn’t used in this crate. There are 6 instances of unused dependencies:
- pallet-subspace
- sc-consensus-subspace-rpc
- sc-consensus-subspace
- subspace-farmer-components
- subspace-service
- subspace-test-client

Reply:
How you spot thoses one ?

cargo clippy do not light on